docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '3.7'
 services:
 
   app:
-    image: daschswiss/dsp-app:latest
+    image: daschswiss/dsp-app:v10.19.2
```

Comment:
who will make sure to keep this up to date? :)

Reply:
we have to, would be nice to have dependabot do it. 'latest' is not published anymore by dsp-app since weeks.